docs/best-practices.md

```diff
@@ -0,0 +1,799 @@
+---
+slug: /best-practices
+---
+
+# Best practices
+
+## Main structure and persistent collections
+
+The main contract structure is marked with `#[near_bindgen]`. It has to be serializable and deserializable with [Borsh](https://borsh.io).
+
+```rust
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct Contract {
+    pub data: String,
+    pub owner_id: AccountId,
+    pub value: u128,
+}
+```
+
+Every time an external method is called, the entire structure has to be deserialized.
+The serialized contract data is stored in [persistent storage] under the key `STATE`.
+
+Change methods ([see below](#view-vs-change-method)) serialize the main contract structure at the end and store the new value into storage.
```

Comment:
these things are already documented, shouldn't any undocumented information just be added to the respective sections?

Reply:
Moved it, adding the small bit about storing in `STATE` which I didn't see elsewhere. Thx again. https://github.com/near/sdk-docs/pull/37/commits/ec84e4162ab37e92f5599b00c1ef31c30897980f